sync_test.go

```diff
@@ -60,7 +60,7 @@ var (
 	// "bd":	OnBlockDisconnected
 	// "fd":	OnFilteredBlockDisconnected
 	wantLog = func() (log []byte) {
-		for i := 796; i <= 800; i++ {
+		for i := 1096; i <= 1100; i++ {
```

Comment:
Curious as to why shifted by 300 blocks now?

Reply:
As we discussed in chat, this is to exercise the checkpoint code.